samples/unit-tests/highcharts/constructors/demo.js

```diff
@@ -139,7 +139,7 @@ QUnit.test('Highstock', function (assert) {
     assert.strictEqual(typeof chart.rangeSelector, 'object', 'Is stock chart');
 
     calledBack = false;
-    chart = Highcharts.StockChart(
+    chart = new Highcharts.StockChart(
```

Comment:
Shouldnt we have some coverage of this and Map without `new`?

Reply:
Yes... You fix?